docs/embeddings.md

```diff
@@ -289,6 +289,30 @@ You can pass in an optional `model_name` argument, which lets you choose which H
 </TabItem>
 </Tabs>
 
+## Jina AI
```

Comment:
Since this embedding function doesn't yet have a JS implementation, it should not appear at all when JS is selected. 

The title and description should be within the python tab. The JS tab is not needed.

Reply:
It seemed to me that there was some linting error when that tab was ignored, but I will check again

Reply:
Check last commit @atroyn , what is wrong there?

Reply:
Sorry, what I meant was that tab can stay empty, but the parts outside the python tab should be moved into it so nothing shows when JS is selected.

Reply:
Ah okey, thanks

Reply:
Done! thanks